### DISCUSSION TITLE
Advanced Alerting Additional Information

### DISCUSSION THREAD

**Discussion by JeffBenedict** [nextgenhealthcare/connect#5217] [Q&A] [answered]
Where can we get additional information on the various topics of Advanced Alerting? The Advanced Alerting User Guide has links to things like Trigger Conditions, Escalation Levels, Action Groups, etc, but those links reference confluence.nextgen.com which we don't seem to have access to.  Are those details in some other document some where?
![2022-05-27_006](https://user-images.githubusercontent.com/106398018/170741534-8fcc1570-158c-4e50-a564-88b64b1ec1db.png)

**Comment by pacmano1** [comment, marked as answer]
Good point - I would open an issue on it.

**Reply by JeffBenedict** [reply]
Thanks. Issue opened: (https://github.com/nextgenhealthcare/connect/issues/5218)
